docs/source/howto/installation.rst

```diff
@@ -385,6 +385,53 @@ Here are a few general tips that might improve the AiiDA performance:
         and try a simple AiiDA query with the new database.
         If everything went fine, you can delete the old database location.
 
+It is not trivial to give a baseline of performance for AiiDA, as it depends a lot on the machine on which it is running, the installation, and what is being run.
```

Comment:
```suggestion
The performance for AiiDA depends both on the hardware it is running on and the processes being run.
```

Reply:
Wonder whether we should give more details here (or at least put it somewhere else). For example what is also important is how/where the services are running. If RabbitMQ and PostgreSQL are running on a different server, that is clearly going to incur a penalty versus them running on the localhost. Same goes for how the type of the filesystem and how it is mounted.

Reply:
That makes a lot of sense. I think I will do that and already add a section in Topics. Will probably be really minimal for now, but at least we'll have a placeholder that we can add onto in the future